suites/quincy/cephadm/tier-1_cephadm-clients.yaml

```diff
@@ -134,6 +134,7 @@ tests:
                 data_devices:
                   all: "true"                         # boolean as string
                 encrypted: "true"                     # boolean as string
+                unmanaged: "true"                     # boolean as string
```

Comment:
Why this change required?

Reply:
So that when any new osd spec is deployed we would not have to manually zap devices which is a lengthy time consuming process. Instead it will automatically assign the OSD's according to the new spec file added below.

Reply:
Also, if we do not apply the unmanaged flag to true then any OSD's if removed would get automatically redeployed for the same service. And any other spec file would not deploy OSD's specific to itself.